src/main/connector/system/WindowsProcess/WindowsProcess.yaml

```diff
@@ -48,32 +47,31 @@ monitors:
             column: 2
             regExp: ${var::matchName}
           - type: keepOnlyMatchingLines
-            column: 11
+            column: 10
             regExp: ${var::matchCommand}
           - type: add
-            column: 14
-            value: $15
+            column: 13
+            value: $14
           - type: divide
-            column: 14 #  unit = 100-nanosecond intervals
+            column: 13 #  unit = 100-nanosecond intervals
             value: 10000000 # divide by 10000000 to get a result in seconds
       mapping:
         source: ${source::processQuery}
         attributes:
-          id: $11 # The commandLine is the monitor id
+          id: $1 # The process id is the monitor id
           process.id: $1
           process.name: $2
           process.parent.id: $3
           process.match.name: ${var::matchName}
           process.match.command: ${var::matchCommand}
-          process.path: $4
         metrics:
-          process.thread.count: $5 # integer
-          process.paging.faults: $6 # integer
-          process.disk.operations{disk.io.direction="read"}: $7 # integer
-          process.disk.operations{disk.io.direction="write"}: $9 # integer
-          process.disk.io{disk.io.direction="read"}: $8 # bytes
-          process.disk.io{disk.io.direction="write"}: $10 # bytes
-          process.memory.usage: $12 # bytes
-          process.memory.virtual: $13 # bytes
-          process.cpu.time: $14 # seconds
-          process.open_file_descriptor.count: $16 # integer
\ No newline at end of file
+          process.thread.count: $4 # integer
+          process.paging.faults: $5 # integer
+          process.disk.operations{disk.io.direction="read"}: $6 # integer
+          process.disk.operations{disk.io.direction="write"}: $8 # integer
+          process.disk.io{disk.io.direction="read"}: $7 # bytes
+          process.disk.io{disk.io.direction="write"}: $9 # bytes
+          process.memory.usage: $11 # bytes
+          process.memory.virtual: $12 # bytes
+          process.cpu.time: $13 # seconds

```

Comment:
Let's add:

```yaml
process.cpu.utilization: rate($13) # ratio
```

So that we have the same metric in both Linux and Windows processes.